XTS/localjunit/WSTX11-interop/src/main/java/com/jboss/transaction/txinterop/test/TestRunner.java

```diff
@@ -119,7 +121,16 @@ else if (testName.startsWith(TestConstants.PREFIX_BA_TESTS))
     private static TestCase createTest(final Class testClass, final String participantURI, final long testTimeout, final boolean asyncTest, final String testName)
         throws IllegalAccessException, InstantiationException, IllegalArgumentException
     {
-        final Object testObject = testClass.newInstance() ;
+        final Object testObject;
+	try {
+	    testObject = testClass.getDeclaredConstructor().newInstance();
+	}
+	catch (InvocationTargetException e) {
+	    throw new RuntimeException( e );
+	}
+	catch (NoSuchMethodException e) {
+	    throw new RuntimeException( e );
+	}
```

Comment:
I think here it is best to add the exceptions to the method signature as createTest is already used in a try/catch block catching Throwable